### PR TITLE
It's not obvious that ascii is an option in this exercise

### PR DESCRIPTION
**Pull request by gilescope**
It's not obvious to the average student that ascii is an option in this exercise. Some people will know this trick, but a lot of people just wouldn't think of it and it makes the exercise more pleasurable. It also makes collecting the results a little nicer at the end too.

Please feel free to suggest changes in the wording!

### CODE REVIEW

File: exercises/practice/minesweeper/.docs/instructions.md [hunk 1]
```diff
@@ -35,3 +35,14 @@ And your code will transform it into this:
 ·2*2·
 ·111·
 ```
+
+## Performance Hint
+
+All the inputs and outputs are in Ascii. Rust `String`s and `&str` are utf8,
+so while one might expect "Hello".chars() to be simple, it actually has to
+check each char to see if it's 1, 2, 3 or 4 `u8`s long. If we know a `&str`
+is ascii then we can call `.as_bytes()` and turn it into a `&[u8]` slice.
```
Comment (gilescope) [inline exercises/practice/minesweeper/.docs/instructions.md:44]: We're not changing the underlying data so maybe word it as:
```suggestion
is ascii then we can call `.as_bytes()` and refer to the underlying data via a `&[u8]` slice.
```

Comment (coriolinus) [reply exercises/practice/minesweeper/.docs/instructions.md:44]: This is clearer, yes.

**Comment by gilescope** [comment]
@coriolinus any other changes needed for approval?

**Review by coriolinus — APPROVED** [review]
Sorry, no; this just fell off my radar with the holidays.

**Pull request merged**